assets/scripts/embedding_tests_dev_0.0.1.min.html

```diff
@@ -39,3 +45,20 @@ <h3>ERROR: 404 error</h3>
   <h3>ERROR: 404 error with deferred loading</h3>
   <oppia oppia-id="fake_id" autoload="false"></oppia>
 </div>
+
+<script type="text/javascript">
+  var currentExplorationId = 'idToBeReplaced';
+  var update = function(){
```

Comment:
Here and elsewhere (multiple places), add a space before {

Reply:
done.